sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -285,10 +285,10 @@ impl Checker for BR02010101Checker {
             return result;
         }
 
-        check_output_contains!(
+        check_output_not_contains!(
             SYSTEMCTL_CMD,
             ["is-active", "chronyd"],
-            &["active"],
+            &["inactive"],
```

Comment:
This would miss the other non-active values in `ActiveState`, like "failed".

You could do the equivalent of:
```
systemctl show --property ActiveState chronyd
```

And check that the output contains `ActiveState=active`.

Reply:
Good call!